packages/dm-core-plugins/src/table/types.ts

```diff
@@ -14,6 +18,7 @@ export type TTablePluginConfig = {
   columns: string[]
   showDelete: boolean
```

Comment:
EditMode and showDelete should be removed from the TTablePluginConfig, as they are no longer part of the blueprint